pkg/log/config.go

```diff
@@ -52,13 +52,56 @@
 package log
 
 import (
+	"fmt"
+	"strings"
+	"time"
+
 	"github.com/natefinch/lumberjack"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 	"go.uber.org/zap/zapgrpc"
 	"google.golang.org/grpc/grpclog"
 )
 
+// This type exists so we can have a bit of logic sitting in front of all the real core calls for the
+// case where we are intercepting other log packages. This lets us subject the log output to the
+// default scope's log level, as is these other log packages were in fact directly outputting to the
+// default scope.
+type interceptor struct {
+	zapcore.Core
+}
+
+func (int interceptor) Check(ent zapcore.Entry, ce *zapcore.CheckedEntry) *zapcore.CheckedEntry {
```

Comment:
Can we rename this to something that is not a type name? `int` shows up as keyword since it's integer type.